server/src/main/java/io/spine/server/integration/DomesticRejectionPublisher.java

```diff
@@ -66,12 +73,18 @@
     public Set<String> dispatch(RejectionEnvelope envelope) {
         final Rejection rejection = envelope.getOuterObject();
         final ExternalMessage message = ExternalMessages.of(rejection, boundedContextName);
-        final ExternalMessageClass messageClass =
-                ExternalMessageClass.of(envelope.getMessageClass());
-        final Publisher channel = publisherHub.get(messageClass);
-        channel.publish(AnyPacker.pack(envelope.getId()), message);
 
-        return ImmutableSet.of(channel.toString());
+        final Iterable<Publisher> channels = router.route(message);
```

Comment:
This portion of code looks a copy-paste changeset from `DomesticEventPublisher`.